src/web-auth/captcha.js

```diff
@@ -201,4 +201,63 @@ function render(auth0Client, element, options, callback) {
   };
 }
 
-export default { render: render };
+/**
+ *
+ * Renders the passwordless captcha challenge in the provided element.
+ *
+ * @param {Authentication} auth0Client The challenge response from the authentication server
+ * @param {HTMLElement} element The element where the captcha needs to be rendered
+ * @param {Object} options The configuration options for the captcha
+ * @param {Object} [options.templates] An object containaing templates for each captcha provider
+ * @param {Function} [options.templates.auth0] template function receiving the challenge and returning an string
+ * @param {Function} [options.templates.recaptcha_v2] template function receiving the challenge and returning an string
+ * @param {Function} [options.templates.recaptcha_enterprise] template function receiving the challenge and returning an string
+ * @param {String} [options.lang=en] the ISO code of the language for recaptcha*
+ * @param {Function} [callback] an optional callback function
+ * @ignore
+ */
+function renderPasswordless(auth0Client, element, options, callback) {
```

Comment:
It looks like `auth0Client` here is the instance of `Authentication`. In a passwordless scenario, would `auth0Client` be an instance of `PasswordlessAuthentication`? (If so, we don't even need a new function here and can re-use `render()` above as-is.

Reply:
Edit: It seems like this function is public so we probably do want to create a separate one.